CHANGELOG.md

```diff
@@ -42,6 +42,47 @@ CLI command and its behaviour. There are no guarantees of stability for the
 
 ### Added
 
+### Changed
+
+### Deprecated
+
+### Removed
+
+### Fixed
+
+### Security
+
+## 2.1.0 - 2023-07-18
+
+After the yanked 2.0.0 release, we're excited to announce our latest major
+version packed with new features and improvements! We've expanded our file type
+recognition, now including Fennel, CommonJS, Qt .pro, .pri, .qrc, .qss, .ui,
+Textile, Visual Studio Code workspace, Application Resource Bundle, Svelte
+components, AES encrypted files, Jakarta Server Page, Clang format, Browserslist
+config, Prettier config and ignored files, Flutter pubspec.lock, .metadata,
+Terraform and HCL, Typst and more.
+
+We've also added the ability to detect SPDX snippet tags in files and introduced
+additional license metadata for the Python package. A new `--json` flag has been
+added to the `lint` command, marking the first step towards better integration
+of REUSE output with other tools.
+
+Notably, `reuse.ReuseInfo` has been enhanced with `copy` and `union` methods and
+now stores information about the source from which the information was gathered.
+
```

Comment:
Because we don't guarantee API stability, I think this can be removed from the message.

Reply:
just removed it